doajtest/example_files/incoming_application.json

```diff
@@ -35,7 +35,7 @@
 		"is_replaced_by": ["2222-2222"],
 		"institution": {
 			"name": "Society Institution",
-			"country": "US"
+			"country": "USA"
 		},
 		"keywords": ["word", "key"],
 		"language": ["EN", "FR"],
```

Comment:
You want to change to 3 letter code for language here?

Reply:
this file seem no one use, but I will also update language for this file.